src/module-elasticsuite-core/etc/config.xml

```diff
@@ -53,6 +53,12 @@
             <term_autocomplete>
                 <max_size>3</max_size>
             </term_autocomplete>
+            <advanced>
+                <extension_enabled>1</extension_enabled>
+                <extension_limit>0</extension_limit>
```

Comment:
Default value should be greater than 0 or the helper method should be reworked.

Reply:
0 should be "no limit" so let's rework the helper method instead.